Client/Frontend/Browser/BrowserLongPressHandler.swift

```diff
@@ -0,0 +1,152 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import UIKit
+
+enum ElementType {
+
+    case ImageSrc(url:NSURL,location:CGRect)
+    case hrefLink(url:NSURL,location:CGRect)
+}
+
+protocol BrowserLongPressHandlerDelegate {
+    func webView(webview: WKWebView, elementsDidLongPressed elements: [ElementType])
```

Comment:
Rather than wrapping all of the WebView methods in delegates, we've been simply adding the delegates directly to the WebView in BrowserViewController (<a href="https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Browser/BrowserViewController.swift#L99">example</a>). Please use the same pattern here -- remove this delegate, and have BVC assign the delegate directly (tab.webView.addGestureRecognizer(...)).